changelog/1576.md

```diff
@@ -0,0 +1,5 @@
+level: major
+reference: issue 1576
+---
+
+AWS Provisioner support has been removed from the UI and it is no longer a navigation menu item.
```

Comment:
```suggestion
AWS Provisioner support has been removed from the UI and it is no longer a navigation menu item.
This service has not been a part of the Taskcluster deployment for some time.
```

Reply:
Oops, I committed these changes twice. It's cool GitHub lets you commit suggestions with a click of a button, but then I wasn't quite sure how to update my fork.